src/version.c

```diff
@@ -64,18 +64,20 @@ init_longVersion(void)
 #else
 	char *date_time = __DATE__ " " __TIME__;
 #endif
-	char *msg = _("%s (%s, compiled %s)");
+	int hp = highest_patch();
+	char *msg = _("%s.%d (%s, compiled %s)");
```

Comment:
I think this will break localized strings. E.g. if you look at src/po/zh_CN.UTF-8.po or src/po/es.po you will see an entry for it. You can test this by doing `LANG=es_ES.UTF-8 vim`.

Reply:
It will, but I'm not sure what the best approach for this is? I suppose I could update all the strings in the .po files(?)

Reply:
@brammool can answer better but I think yes, you have to update all the .po files. They have individual owners though, but I'm not sure if a change like this which are more programmatic in nature than changing the texts need a review (I would guess not).